v1.0/brainAtlasVersion.schema.json

```diff
@@ -14,68 +14,71 @@
   "properties": {
     "alias": {
       "type": "string",
-      "description": "User-defined identifier, often a short name, of this brain atlas version."
+      "_instruction": "Enter the user-defined identifier for this brain atlas version (e.g. a short name)."
     },
     "alternativeVersionOf": {
       "type": "array",
-      "description": "Link to an alternative brain atlas versions.",
+      "_instruction": "Add one or several alternative versions of this brain atlas versions.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/BrainAtlasVersion"
-      ]
+      ],
+      "minItems": 1,
+      "uniqueItems": true
     },
     "annotationSet": {
       "type": "array",
-      "description": "List of annotations that define this brain atlas version",
+      "_instruction": "Add one or several annotations included in this brain atlas version.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/Annotation"
-      ]
+      ],
+      "minItems": 1,
+      "uniqueItems": true
     },
     "coordinateSpace": {
-      "type": "object",
-      "description": "Link to coordinate space this brain atlas version is represented in",
+      "_instruction": "Add the coordinate space in which this brain atlas version exists in.",
       "_linkedTypes": [
         "https://openminds.ebrains.eu/sands/CoordinateSpace"
       ]
     },
     "fullName": {
       "type": "string",
-      "description": "A descriptive name of this brain atlas."
+      "_instruction": "Enter a descriptive name for this brain atlas version."
     },
     "newVersionOf": {
-      "description": "Link to the previous brain atlas version.",
+      "_instruction": "Add the earlier version this brain atlas version is a new version of.",
```

Comment:
@aeidi89 and I talked about this and she found this simplified description a bit confusing. I think it was because new brain atlas versions may not originate from the one before that one (e.g. v3 could be new from v2 which could be new from v1, but it could also be that v3 and v2 both are new ones from v1 (but too different to be alternative versions or called v2.0 and v2.1), correct @aeidi89? 
If I understood the argumentation wrong, please correct me :)

Reply:
@UlrikeS91 , yes, you summarized it well :)

Reply:
Conclusion with @lzehl: "Add the earlier version of this brain atlas version." (Earlier vs previous: earlier is fine, but instruction was weird and too long.)